dotcom-rendering/src/web/layouts/CommentLayout.tsx

```diff
@@ -407,7 +407,7 @@ export const CommentLayout = ({ CAPIArticle, NAV, format }: Props) => {
 							cssOverrides={css`
 								display: block;
 							`}
-							color={palette.border.article}
+							color={palette.border.secondary}
```

Comment:
I had to go with `palette.border.secondary` for `StraightLines` because `palette.border.article` was breaking Labs straight lines.